apps/mobile/src/app/components/Auction/BidModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Modal } from 'react-native';
+import { bidModalStyles as styles } from './bid-styles';
+import { Text, View, Button } from '@inheartive/ui/atoms';
+
+interface BidModalProps {
+  closeModal: () => void;
+  bid: number;
+}
+
+export const BidModal = ({ closeModal, bid }: BidModalProps) => (
+  <View style={styles.centeredView}>
```

Comment:
Cool modal, maybe it would be good togeneralize it a bit and move it to our library?